### DISCUSSION TITLE
Dex version update from 2.32.0

### DISCUSSION THREAD

**Discussion by dsetlock** [terrakube-io/terrakube#570] [General]
I noticed that the Dex IDP version is set at 2.32.0, which seems to be about 3 years behind. Should I modify the version for deployment to accommodate recently added features in Dex, or is there a planned update cycle for this? 

Or, am I completely misinterpreting this?

**Comment by alfespa17** [comment]
Hello @dsetlock 

I think we need to update the documentation and remove the value  2.32.0

We are using dex helm chart as a dependency as you can see here:

https://github.com/AzBuilder/terrakube-helm-chart/blob/main/charts/terrakube/Chart.yaml#L37

In Dex helm chart version 0.13.0 there is a propery "tag" that can be use to override the dex version

https://github.com/dexidp/helm-charts/blob/dex-0.13.0/charts/dex/values.yaml#L20

So when you deploy the terrakube helm chart you could use something like this:

```yaml
dex:
  image:
    tag: X.X.X
  config:
     ........
```

You should be able to use a newer version, you only need to change the dex image tag as the example above and it should work

**Reply by alfespa17** [reply]
By default it looks like the dex helm chart 0.13.0 is using dex version v2.35.3

https://github.com/dexidp/helm-charts/blob/dex-0.13.0/charts/dex/Chart.yaml#L5
